octoprint_mrbeam/static/js/convert.js

```diff
@@ -845,17 +878,26 @@ $(function () {
                 self.filled_shapes_placed()
             );
         });
-        self.imgIntensityWhite = ko.observable(0);
-        self.imgIntensityBlack = ko.observable(50);
-        self.imgFeedrateWhite = ko.observable(1500);
-        self.imgFeedrateBlack = ko.observable(250);
-        self.imgDithering = ko.observable(false);
-        self.beamDiameter = ko.observable(0.15);
-        self.engravingPiercetime = ko.observable(0);
-        self.engravingCompressor = ko.observable(0);
-
-        self.sharpeningMax = 25;
-        self.contrastMax = 2;
+        self.imgIntensityWhite = ko.observable(
+            self.JOB_PARAMS.default.intensityWhite
+        );
+        self.imgIntensityBlack = ko.observable(
+            self.JOB_PARAMS.default.intensityBlack
+        );
+        self.imgFeedrateWhite = ko.observable(
+            self.JOB_PARAMS.default.feedrateWhite
+        );
+        self.imgFeedrateBlack = ko.observable(
+            self.JOB_PARAMS.default.feedrateBlack
+        );
+        self.imgDithering = ko.observable(self.JOB_PARAMS.default.dithering);
+        self.beamDiameter = ko.observable(self.JOB_PARAMS.default.beamDiameter);
+        self.engravingPiercetime = ko.observable(
+            self.JOB_PARAMS.default.pierceTime
+        );
+        self.engravingCompressor = ko.observable(
+            self.JOB_PARAMS.default.engCompressor
+        );
```

Comment:
refering to earlier, you can set the settings as such:
```
[...]
self.settingsViewmodel = params[1]
[...]

self.onBeforeBinding = function () {
    self.settings = self.settingsViewModel.settings.plugins.mrbeam.job;
};
```
And then use `self.setttings.default.intensityBlack()`